File: webui/src/main/scala/com/taskadapter/webui/pages/DevPageFactory.scala
package com.taskadapter.webui.pages

import com.taskadapter.webui.export.ExportResultsFragment
import com.taskadapter.webui.results.ExportResultFormat
import com.vaadin.flow.component.Component


/**
  * sample code for faster development.
  */
object DevPageFactory {
  def getDevPage(result: ExportResultFormat): Component = {
    val page = new ExportResultsFragment(false)
    page.showExportResult(result)
  }
}

File: webui/src/main/java/com/taskadapter/webui/pages/DevPageFactory.java
package com.taskadapter.webui.pages;

import com.taskadapter.webui.export.ExportResultsFragment;
import com.taskadapter.webui.results.ExportResultFormat;
import com.vaadin.flow.component.Component;

/**
 * sample code for faster development.
 */
public class DevPageFactory {
    public static Component getDevPage(ExportResultFormat result) {
        var page = new ExportResultsFragment(false);
        return page.showExportResult(result);
    }
}
